includes/data-port/export-tasks/class-sensei-export-task.php

```diff
@@ -125,6 +125,10 @@ private function get_serialized_post( $post ) {
 
 		return array_map(
 			function( $column ) use ( $columns ) {
+				if ( ! isset( $columns[ $column ] ) ) {
```

Comment:
We should probably have this anyway, but this is until we remove the lesson field from the import schema.